### DISCUSSION TITLE
"Pattern of mutations" meaning in log_mutation_likelihood

### DISCUSSION THREAD

**Discussion by dennisprangle** [tskit-dev/msprime#2324] [Q&A] [answered]
The log_mutation_likelihood documentation says it 

> Returns the unnormalised log probability of the stored pattern of mutations on the stored tree sequence, assuming infinite sites mutation.

Does "pattern of mutations" mean (1) the locations of the mutations in the ARG or (2) the sets of mutations at each leaf? If it's (1), is there an easy way to get the likelihood for (2)?

**Comment by JereKoskela** [comment, marked as answer]
It's (2).
